terraform/aws/variables.tf

```diff
@@ -44,17 +44,23 @@ variable "user_buckets" {
 }
 
 variable "hub_cloud_permissions" {
-  type        = map(object({ requestor_pays : bool, bucket_admin_access : set(string), extra_iam_policy : string }))
+  type = map(object({
+    allow_access_to_requestor_pays_buckets : optional(bool, false),
```

Comment:
Turns out we actually never implemented this on AWS! So we should just remove this from here.